src/main/java/teammates/storage/entity/CourseStudent.java

```diff
@@ -20,7 +20,14 @@
  * [enrolled in] --> Course.
  */
 @PersistenceCapable
-public class CourseStudent implements StoreCallback {
+public class CourseStudent extends Entity implements StoreCallback {
+
+    /**
+     * The name of the primary key of this entity type.
+     */
+    @NotPersistent
+    public static final String PRIMARY_KEY_NAME = getFieldWithPrimaryKeyAnnotation(CourseStudent.class);
+
```

Comment:
There are some changes to existing entity classes. Confirm new code works with legacy data?